README.md

```diff
@@ -1,3 +1,22 @@
+# Spectrometer
+
+Spectrometer is a polyglot dependency analysis tool, designed to identify and
+report dependency trees for many languages and package managers.  See the
+[docs](docs/strategies.md) for a non-exhaustive list of supported
+languages/managers.
+
+```sh
+me@mydesk:~/myproject$ fossa2 analyze -o
```

Comment:
I would use the quick-start to demonstrate an analysis upload. Most people will read this snippet and just run it without reading anything else, and providing the output snippet first will confuse them.

Reply:
If a potential user that does not use the full FOSSA suite tries that, it won't work.  That behavior in a quickstart section would make me seriously reconsider using any tool published by that user/org.

Reply:
I'll continue that logic here: 

The point of a quickstart should be "How do I check that this thing works?", not "Now that I've finished my other setup, how does this piece fit in?".  That belong in the user manual, and for our product, it belongs at the top of the user manual, since that is really what we're offering/selling.

I think a good analog to this is setting up a database server.  The quickstart there would not include things like security, HighAvailability, sharding, etc.  That's all critical for running in the real world, and should be documented well, but it's not the first thing you want to see during your first experiment with the tool.

Reply:
For almost all users, the first thing they want to do is upload an analysis. We can provide the output snippet later down, but the first line in the quickstart should show them how to upload an analysis.

Reply:
Ok, I have an idea how these can coexist.  I can make a small quickstart section that has both options and starts with the upload bit, but is prefaced with "If you have an API key".

I think we're underplaying the "I have literally nothing else set up yet" scenario, and in that case, we're responsible here for making a great first impression.

Reply:
> I think we're underplaying the "I have literally nothing else set up yet" scenario, and in that case, we're responsible here for making a great first impression.

Can you tell me more about why this is a prominent scenario?

In the last 2 years I've worked on this project, people almost never care about integrating their own code directly against the output. Generally, people only care about using this together with the service. In fact, I wouldn't even preface with "if you have an API key" - I would add a link to the web application docs and direct people to create an API key if they don't have one.

For comparison, think about Snyk's CLI or GitHub's CLI. Would you expect to use those _without_ integrating with the web application? The output mode is really only for serious power users, who can figure it out themselves without needing a quick start (besides, we'll write a power user guide later anyway).

Reply:
I'm less concerned about power users and more concerned about someone whose first interaction with FOSSA is through this project, and that leads them to other FOSSA services.  But I'm OK to defer it to a power user guide.  And the historic context is a part I definitely haven't seen much of.  I'll shift it the way you're suggesting.